nightly/tests_for_nayduck.txt

```diff
@@ -51,8 +51,10 @@ pytest contracts/deploy_call_smart_contract.py
 pytest --timeout=240 contracts/gibberish.py
 
 # python stress tests
-# pytest --timeout=2000 stress/stress.py 3 3 3 0 staking transactions local_network
+pytest --timeout=2000 stress/stress.py 3 3 3 0 staking transactions
+pytest --timeout=2000 stress/stress.py 3 3 3 0 staking transactions local_network packets_drop
```

Comment:
`local_network` without `packets_drop` doesn't provide much additional coverage.
`node_restart` on the other hand has major difference with and without `packets_drop`: in the former case the nodes are not proxified, and thus the timing is rather different, so keeping both `node_restart` and `node_restart packets_drop` in the default set.